### DISCUSSION TITLE
Need documentation to setup tensorflow in Photoprism.

### DISCUSSION THREAD

**Discussion by doctorttt** [photoprism/photoprism#3507] [General]
Hi, I researched this topic on google and couldn't find much info about. I heard that you can use pre-configured models to do facial recognition, etc. A couple questions:

1. Is Photoprism's tensorflow a sponsor-only feature? 
2. Is there any documentation explaining how to setup tensorflow in Photoprism? I have a latest Nvidia gpu that has CUDA and tensor cores. I just couldn't find any documentation anywhere even on Photoprism's website. Am I missing something? Also, nobody really talks about tensorflow on this github? I guess it's not a feature that people want? 
3. Do I need to use a special docker image? or ```photoprism/photoprism``` would work? 
4. Does the Photoprism's image include the necessary tensorflow libraries? Or User will need to pass certain libraries from the host to the container?

**Comment by doctorttt** [comment]
no help at all? then what's the point of having tensorflow on the photoprism webpage without any documentation?

**Reply by lastzero** [reply]
It's not a member feature just like 99% of the features our software includes. See the overview on our website. We provide pre-built libraries on dl.photoprism.app. You can also build TensorFlow from source, but it may take a week if you are skilled. Help with this is not included in our free Community support.
